app/views/recipes/index.html.erb

```diff
@@ -3,12 +3,13 @@
     <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
   <% end %>
 
-  <div class="flex justify-between items-center">
-    <h1 class="font-bold text-4xl">Recipes</h1>
+  <div class="flex justify-center items-center">
     <%= link_to "New recipe", new_recipe_path, class: "rounded-lg py-3 px-5 bg-blue-600 text-white block font-medium" %>
```

Comment:
Yea, I didn't put a lot of thought on the navigation links. I based it on what most websites have, I just wanted to get it to work. I can remove the home link, and leave the create recipe link. I'll remove the button as well. Right now since we don't have a lot of routes, I'm unsure what other links to add to nav bar. I'm sure at some point we will have a login/out link, maybe user profile etc. Unless you have some ideas we can add right now.

Reply:
Nope no ideas for anything at the moment which I think is fine